packages/components/src/mobile/bottom-sheet/cycle-picker-cell.native.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { findIndex } from 'lodash';
+/**
+ * Internal dependencies
+ */
+import Cell from './cell';
+
+export default function BottomSheetCyclePickerCell( props ) {
+	const {
+		value,
+		options,
+		onChangeValue,
+		...cellProps
+	} = props;
+
+	const cycleOptionValue = () => {
+		return options[ ( findIndex( options, [ 'value', value ] ) + 1 ) % options.length ].value;
+	};
+
+	return (
+		<Cell
+			onPress={ () => onChangeValue( cycleOptionValue() ) }
+			editable={ false }
+			value={ options[ findIndex( options, [ 'value', value ] ) ].label }
```

Comment:
Same here, let's create variables with explicit names that should help improve readability.

Reply:
See comment above, this was improved in 9aa8aa7d5d5aad5592b40096f6ddb6e1d5c19138 and 12ebc85ea833e3ebce644bffdf7273448c12151d